src/actions.tex

```diff
@@ -631,40 +621,45 @@
 \begin{itemize}
 	\item Must Own 1+ Province in Area(s) governed by the Religion slot
 	\item All Provinces must be owned by Realms of the same State Religion
+	\item If Protestant, pay -1\adminpower (p.~38)
 	\item Change Religion to your State Religion
 	\item Add 1\unrest to your affected \towns
-	\item Roll Rebel Dice for all your \unrest in affected Area(s)
+	\item Roll Rebel Dice in affected Area(s)
 \end{itemize}
 
 \subsectionstriped{colorDiplo}{Diplomatic Actions}
-\actionHeadingNoSpace{Influence \normal{(1\diplopower/3\ducats per \influence) (p.~15)}}
-\begin{itemize}
-	\item Pay 1\diplopower or 3\ducats per \influence (min 1\diplopower)
-	\item May place \influence in Areas Adj. to your \towns, \vassals, \alliances, \marriages, \influence that were there at the start of the Turn
-	\item Max 2\influence per Area per Turn
-	\item Area may contain max 5\influence
-	\item No \influence on Distant Continents
-\end{itemize}
-
 \actionHeading{Forge Alliance \normal{(1-3\diplopower) (p.~15)}}
 \begin{itemize}
 	\item Only with an NPR
 	\item Both must be at Peace
 	\item Must have at least 2\influence in target Realm
-	\item If target is a Distant NPR, then must have a \claim on 1+ of target's Areas
+	\item If target is a Distant NPR, then 
+	\begin{itemize}
+		\item Must have a \claim on 1+ of target's Areas
+		\item Use \colonist instead of \influence (p.~32)
+	\end{itemize}
 	\item If target has Alliance, then must have more \influence and remove previous \alliance
 	\item Pay \diplopower = ½ of target's Tax Inc. (max 3)
 	\item Add \alliance to target's Capital
-	\item Add 1\influence to target's Cap. Area (if not full)
+	\item Add 1\influence to target's Cap. Area
+\end{itemize}
+
+\actionHeadingNoSpace{Influence \normal{(1\diplopower/3\ducats per \influence) (p.~15)}}
+\begin{itemize}
+	\item Pay 1\diplopower or 3\ducats per \influence (min 1\diplopower)
+	\item May place \influence in Areas Adj. to your \towns, \vassals, \alliances, \marriages, \influence that were there at the start of the Turn
+	\item Max 2\influence per Area per Turn
+	\item No \influence on Distant Continents
```

Comment:
Seems like there is enough room to put back "Area may contain max 5\influence" clarification